types/lib/amazon-pay.d.ts

```diff
@@ -0,0 +1,34 @@
+import { Emitter } from './emitter';
+
+export type AmazonPayOptions = {
+  /**
+   * 2 Digit Country Code
+   */
+  region: string;
+
+  /**
+   * The customer's locale. This is used to set the language rendered in the UI.
+   */
+  locale: string;
+
+  /**
+   * The currency of the payment.
+   */
+  currency: string;
```

Comment:
Shouldn't all of these options be optional like you have set `gatewayCode` to be?

Reply:
Actually, it looks like `currency` and `locale` are set based on the `region`. They should not be included as options.

Reply:
It also looks like there's a `sandbox` option that gets used in `renderButton`. I think that should be included as well as an optional option.